lib/themes/dosomething/paraneue_dosomething/js/components/CampaignSlide.js

```diff
@@ -13,7 +14,30 @@ class CampaignSlide extends React.Component {
     this.controller = new CampaignController();
   }
 
+  componentDidMount() {
+    // get campaigns, this.controller.getCampaigns({
```

Comment:
OH, yeah I was just writing out the idea we need to get campaigns here, wasn't trying to write anything yet